app/views/courses/_campaign_courses.html.haml

```diff
@@ -1,7 +1,10 @@
 %section#courses
   .section-header
-    %h3= [t("#{presenter.course_string_prefix}.courses"), presenter.campaign.title].join(" | ")
-    .sort-select.inset
+    %h3
+      = t("#{presenter.course_string_prefix}.courses")
+      - if current_user && (current_user.admin? || Features.open_course_creation?)
```

Comment:
Since we're using a presenter already, this logic should be done as a clearly named method in the presenter.